### ISSUE TITLE
processingLogAppend() using local time, but should be UTC

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1587]
Below, from a `summary()` on an `oceanglider` object) demonstrates. Note that the final two comments (where I am testing the keypress flow code) are giving processingLog times of 10:21, which is my local time, instead of the sometime after 13:00, which is UTC.

PS. Somewhat confusingly, I have double times in those last two events. That's because I am saving all the comments into the processing log at the time of saving the object, which makes sense because some comments might get written when the file is loaded, others during processing, and others before saving ... this is a choice best left to the operator.

```
Processing Log
    - 2019-07-30 14:06:51.304 UTC: `create 'glider' object`
    - 2019-07-30 14:06:51.304 UTC: `initializeFlagScheme(object, name="IOOS", mapping=list(pass=1,not_evaluated=2,suspect=3,fail=4,missing=9)), default=c()`
    - 2019-07-30 14:08:18.995 UTC: `read.glider.seaexplorer.delayed(directory="~/Dropbox/data/glider/delayedData/SEA021/Data/M51/pld/logs", yo=1:943, level=1)`
    - 2019-08-06 10:21:27.349 UTC: `[2019-08-06 13:20:10.161] Danny`
    - 2019-08-06 10:21:27.349 UTC: `[2019-08-06 13:21:19.029] Does p get skipped? yes! mooo`
```

**Comment by dankelley** [comment]
Hm, this is pretty weird because `R/processingLog.R` has as below, which is using `presentTime()`, the very same function I am using in `oceanglider` to get the times listed in the square brackets above (the 13:20 times).  Maybe there is some kind of problem with environment variables.

```R
processingLogAppend <- function(h, value="")
{
    if (inherits(h, "oce"))
        h <- h@processingLog
    res <- if (is.null(h)) list(time=NULL, value=NULL) else h
    if (is.null(h$time[1])) {
        res$time <- presentTime()
        res$value <- value
    } else {
        res$time <- c(res$time, presentTime())
        res$value <- c(res$value, value)
    }
    res
}
```

**Comment by dankelley** [comment]
Oh, the times are ok. The problem is that `summary` is not formatting to UTC, e.g. 
```R
> format(ctd@processingLog$time[4])
[1] "2019-08-06 10:37:04.688"
> format(ctd@processingLog$time[4],tz="UTC")
[1] "2019-08-06 13:37:04.688"
```

**Comment by dankelley** [comment]
This is fixed in commit bc4793a14da135060bfa081c3f9610e7b87c7d6d of the "develop" branch, as exemplified by:
```R
> example("processingLog<-")

prcL<-> data(ctd)

prcL<-> processingLogShow(ctd)
* Processing Log
    - 2018-11-14 20:03:47.049 UTC: `create 'ctd' object`
    - 2018-11-14 20:03:47.093 UTC: `read.ctd.sbe(file = file, debug = 10, processingLog = processingLog)`
    - 2018-11-14 20:03:47.102 UTC: `oce.edit(x = ctd, item = "startTime", value = as.POSIXct(gsub("1903",     "2003", format(ctd[["startTime"]])), tz = "UTC") + 4 * 3600,     reason = "file had year=1903, instead of 2003", person = "Dan Kelley")`

prcL<-> processingLog(ctd) <- "test"

prcL<-> processingLogShow(ctd)
* Processing Log
    - 2018-11-14 20:03:47.049 UTC: `create 'ctd' object`
    - 2018-11-14 20:03:47.093 UTC: `read.ctd.sbe(file = file, debug = 10, processingLog = processingLog)`
    - 2018-11-14 20:03:47.102 UTC: `oce.edit(x = ctd, item = "startTime", value = as.POSIXct(gsub("1903",     "2003", format(ctd[["startTime"]])), tz = "UTC") + 4 * 3600,     reason = "file had year=1903, instead of 2003", person = "Dan Kelley")`
    - 2019-08-06 13:51:13.457 UTC: `test`
```
